src/components/Card.js

```diff
@@ -0,0 +1,47 @@
+import noneCardImg from '../assets/Linkbrary.svg';
+import kebab from '../assets/kebab.svg';
+import emptyStar from '../assets/icon-empty-star.svg';
+import fullStar from '../assets/icon-full-star.svg';
+import ElapsedTime from '../utils/ElapsedTime';
+import FormatDate from '../utils/FormatDate';
+import './Card.css';
+
+const Card = ({ link }) => {
+  const { created_at, description, image_source, url } = link;
+  const fillStar = false;
+
+  return (
+    <div className="card_area">
+      <a
+        href={url}
+        className="card_link"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        <div
+          className={image_source ? 'img_area' : 'img_area none_img_area'}
```

Comment:
전에 멘토링 때 말씀드린 부분이랑 유사한데, 변경되는 부분만 조건부 처리하는 게 좋습니다.
```suggestion
          className={`img_area${image_source ? '': ' none_img_area'}`}
```